src/qml/SearchBar.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick
+import QtQuick.Controls
+import org.qfield
+import Theme
+
+Rectangle {
+  z: 20
```

Comment:
why adding a z value here?

Reply:
previously it was in Listview header but now we can remove it :) , Done.